test/recovery_to_vault.js

```diff
@@ -134,7 +145,7 @@ contract('Recovery to vault', accounts => {
 
   // Test both the Kernel itself and the KernelProxy to make sure their behaviours are the same
   for (const kernelType of ['Kernel', 'KernelProxy']) {
-    context(`> ${kernelType}`, () => {
+    context.only(`> ${kernelType}`, () => {
```

Comment:
Why are we adding `.only` here? Are we trying to skip something?

Reply:
Fixed in https://github.com/aragon/aragonOS/pull/483